quickwit/quickwit-storage/src/object_storage/s3_compatible_storage.rs

```diff
@@ -643,7 +675,7 @@ impl S3CompatibleObjectStorage {
                 }
                 Err(delete_objects_error) => {
                     error = Some(delete_objects_error.into());
-                    unattempted.extend(chunk.iter().map(|path| path.to_path_buf()));
+                    unattempted.extend(chunk.iter().map(|(path, _oid)| path.to_path_buf()));
```

Comment:
This is technically incorrect. These path have been attempted. 
For instance, at this point, they might be deleted.